From e788ea3c8c74a68bfc36fc4d9b7ab9ec7009d21d Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Thu, 9 Aug 2012 10:55:20 +0800
Subject: ENGR00215952 HDMI:'PHY PLL not locked' messages during boot

If using mxcfb1 for HDMI display, it will print
'PHY PLL not locked'.
Fixed it with setting HDMI default to blank state.

Signed-off-by: Sandor Yu <R01008@freescale.com>
---
 drivers/video/mxc_hdmi.c |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/video/mxc_hdmi.c b/drivers/video/mxc_hdmi.c
index e53301d..87a1af1 100644
--- a/drivers/video/mxc_hdmi.c
+++ b/drivers/video/mxc_hdmi.c
@@ -2113,6 +2113,9 @@ static int mxc_hdmi_disp_init(struct mxc_dispdrv_handle *disp,
 
 	hdmi->dft_mode_set = false;
 
+	/* Setting HDMI default to blank state */
+	hdmi->blank = FB_BLANK_POWERDOWN;
+
 	setting->dev_id = mxc_hdmi_ipu_id;
 	setting->disp_id = mxc_hdmi_disp_id;
 	setting->if_fmt = IPU_PIX_FMT_RGB24;
-- 
1.7.9.5

